dist2src/core.py

```diff
@@ -464,6 +464,34 @@ def perform_convert(self, origin_branch: str, dest_branch: str):
         # get all the patch-commits
         self.rebase_patches(from_branch=TEMP_SG_BRANCH, to_branch=dest_branch)
 
+    def apply_import_tag(self, origin_branch: str, dest_branch: str):
+        """
+        Find the 'import' tag in dist-git at the tip of origin_branch
+        and apply it to the tip of dest_branch.
+        """
+        dist_git_head = self.dist_git.repo.heads[origin_branch].commit
+        import_tags = [
+            tag
+            for tag in self.dist_git.repo.git.tag("--points-at", dist_git_head).split(
+                "\n"
+            )
+            if tag.startswith("imports/")
+        ]
+        if len(import_tags) > 1:
+            raise RuntimeError(
+                f"More than one import tag found on commit {dist_git_head}: {import_tags}"
+            )
+        if not import_tags:
+            raise RuntimeError(f"No import tags found on commit {dist_git_head}")
```

Comment:
I wonder what would be the resolution when this happens

Reply:
I've changed the method, as it turns out not all branches in dist-git have an import tag at the tip of the branch.

Instead we are creating our own tag using the name of the dist-git branch and the hash of the commit at the tip of that branch (ex: `convert/c8s/05262ab9019701b9d278b305017861200204f559`) and applying that one in source-git.

See and example [here](https://git.stg.centos.org/fork/csomh/source-git/acl/commits/c8).